src/crypto/OlmDevice.js

```diff
@@ -81,6 +81,17 @@ function OlmDevice(sessionStore, cryptoStore) {
     this.deviceEd25519Key = null;
     this._maxOneTimeKeys = null;
 
+    // track which of our other devices (if any) have cross-signed this device
+    // XXX: this should probably have a single source of truth in the /devices
+    // API store or whatever we use to track our self-signed devices.
+    this.crossSelfSigs = [];
```

Comment:
this interloper doesn't seem to be being used?

Reply:
presumably popping in from the cross signing stuff :)